### DISCUSSION TITLE
Ubuntu 22.04.1 LTS Upgrade - Problems?

### DISCUSSION THREAD

**Discussion by gsownsby** [searxng/searxng#2147] [Q&A] [answered]
Hello All,
I have a new SearXNG installation up and running on Linode.com.  I see prompts to upgrade to Ubuntu 22.04.1 LTS.  Any issues with doing so?  Thank you.

Gary

**Comment by return42** [comment, marked as answer]
You can upgrade .. but your question reminds me, that I have to add ubu2204 to the list of LXC images.

https://github.com/searxng/searxng/blob/9d102fb08ff4a7dc04d093561adde60436e3f69e/utils/lxc-searxng.env#L19-L20 

I added ubu2204 to the LXC images and tested the installation scripts:

- https://github.com/searxng/searxng/pull/2148

**Reply by dalf** [reply]
One thing about Ubuntu 22.04 : `make test.robot` (and `make test`) is super slow because Firefox is installed with an Ubuntu snap. 
But I guess you just deploy SearXNG: there are no issue with Ubuntu 22.04 AFIK.

**Reply by return42** [reply]
> because Firefox is installed with an Ubuntu snap.

Wow, couldn't believe when I readded ..  but you are right; they install/maintain firefox only in snap .. what a shame ..

- ubu2204: https://packages.ubuntu.com/jammy/firefox and before:
- ubu2004: https://packages.ubuntu.com/focal/firefox
